app/models/user.rb

```diff
@@ -3,10 +3,14 @@ class User < ApplicationRecord
   has_many :groups, through: :user_groups
   has_many :lists
   has_many :invitations
+  has_many :invitations_received, foreign_key: :receiver_id, class_name: 'Invitation'
+  has_many :invitations_sent, foreign_key: :sender_id, class_name: 'Invitation'
+  has_many :user_exclusion_teams
+  has_many :exclusion_teams, through: :user_exclusion_teams
+  has_many :secret_santa, foreign_key: :santa_id, class_name: 'SantaAssignment'
+  has_many :santa_recipient, foreign_key: :receiver_id, class_name: 'SantaAssignment'
```

Comment:
for the `has_many :secret_santa` and `has_many :santa_recipient` relations, I think the more semantic choice is `:secret_santas` and `:santa_recipients`. Do you agree?